### PR TITLE
ADEN-10720 Set fixed tlb height on selected wikis

### PR DESCRIPTION
**Pull request by fraszczakszymon**
## Links

* http://wikia-inc.atlassian.net/browse/ADEN-10720

### CODE REVIEW

**Review by dmnsobczak — APPROVED** [review]

File: app/modules/ads/slots.js [hunk 1]
```diff
@@ -462,6 +464,43 @@ export const slots = {
...
+          adSlot.addClass('wrapper-gap-disabled'); // shrink with animation
+        } else {
+          const id = scrollListener.addCallback(() => {
+            if (!universalAdPackage.isFanTakeoverLoaded()) {
+              adSlot.addClass('wrapper-gap-disabled'); // shrink with animation
+
+              scrollListener.removeCallback(id);
+            }
+          });
+        }
```
Comment (Loczi94) [inline app/modules/ads/slots.js]: This part is completely unreadable for me. 🙈 
What do you think about creating separate methods like `shrinkWithouAnimation` and `shrinkWithoutAnimation` so there is no need to leave comments in code?
Also maybe we could move if statement from line 490 and make else if statement in 488?

File: app/templates/components/ad-slot.hbs [hunk 2]
```diff
@@ -1,6 +1,6 @@
 <div
   id={{name}}
-  class="ad-slot {{adEngine3ClassName}} {{hiddenClassName}}"
+  class="ad-slot {{adEngine3ClassName}} {{defaultClasses}}"
```
Comment (Loczi94) [inline app/templates/components/ad-slot.hbs:3]: Will it work with an array?

File: tests/integration/components/article-content-test.js [hunk 3]
```diff
@@ -90,11 +90,6 @@ module('Integration | Component | article content', (hooks) => {
       }}{{/article-content}}`);
 
     assert.dom(topLeaderboardSelector).exists({ count: 1 });
-    assert.equal(
-      find(topLeaderboardSelector).previousSibling,
-      find('.wiki-page-header'),
-      'previous element is site header',
-    );
```
Comment (Loczi94) [inline tests/integration/components/article-content-test.js]: What's wrong with this test?

File: app/templates/components/ad-slot.hbs (hunk 2 above)
Comment (fraszczakszymon) [reply app/templates/components/ad-slot.hbs:3]: Nope. This is why there is join(' '): https://github.com/Wikia/mobile-wiki/pull/2409/files#diff-c158578aa157904c3f0096a4f978d60017581fbd5338bb9ebf9b06fd40b0d1d6R195

File: tests/integration/components/article-content-test.js (hunk 3 above)
Comment (fraszczakszymon) [reply tests/integration/components/article-content-test.js]: Thanks for catching. It was removed at the time when top-leaderboard had additional wrapper. It's no longer the case. Assertion reverted.

**Review by Loczi94 — APPROVED** [review]

**Review by nandy-andy — APPROVED** [review]

**Pull request merged**